src/Caching/StackExchangeRedis/src/StackExchangeRedisCacheServiceCollectionExtensions.cs

```diff
@@ -12,6 +13,25 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// </summary>
 public static class StackExchangeRedisCacheServiceCollectionExtensions
 {
+    /// <summary>
+    /// Adds Redis distributed caching services to the specified <see cref="IServiceCollection" />.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+    /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+    public static IServiceCollection AddStackExchangeRedisCache(this IServiceCollection services)
```

Comment:
Can redis be used with only default settings? I doesn't look like it. This overload shouldn't exist if settings are required.